### PR TITLE
Add parameter to force checking all integrations

### PR DESCRIPTION
**Pull request by mrodm**
## What does this PR do?

Add parameter to allow checking all the integrations independently of the latest commits. This will be used mainly to run the schedule-daily job.

### CODE REVIEW

File: .ci/Jenkinsfile [hunk 1]
```diff
@@ -272,6 +274,11 @@ def isPrAffected(integrationName) {
     }
   }
 
+  if (env.FORCE_CHECK_ALL == "true") {
+    echo "[${integrationName}] PR is affected: \"force_check_all\" parameter enabled"
+    return true
+  }
```
Comment (jsoriano) [inline .ci/Jenkinsfile:280]: This could be moved to the top of the function, before reading the manifest, right?

Comment (mrodm) [reply .ci/Jenkinsfile:280]: I wanted to keep the conditions regarding kibana.version, so if the package version does not support the given Elastic Stack it keeps returning `false`.

Comment (jsoriano) [reply .ci/Jenkinsfile:280]: Ah, you are right :+1:

**Comment by mrodm** [comment]
Merging branch, CI failures are unrelated to this change, and the integrations that failed on one build were successful on the following or viceversa.

**Pull request merged**